test/wallet-accounts-reveal-test.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-env mocha */
```

Comment:
These test are awesome! QQ about tests around auctions:

Is there a "standard" test setup for testing cases at different phases of an auction?

If not - #168 introduces what could be a good starting (at least for discussion?): https://github.com/handshake-org/hsd/pull/168/files#diff-bd762abfcd16ee1dedbbbe11817ebec1. It sets up an auction, moves it through different phases with nested `describe` blocks.

Reply:
This is a good reference, but it operates on HTTP API endpoints and `hs-client` so it is kind of more of an integration test. I'm trying to fix a bug in the `wallet` module here, so I think it makes sense to just isolate that unit.